publish.rs

```diff
@@ -216,6 +216,6 @@ fn publish(krate: &Crate) {
         .status()
         .expect("failed to run cargo");
     if !status.success() {
-        println!("FAIL: failed to publish `{}`: {}", krate.name, status);
+        panic!("FAIL: failed to publish `{}`: {}", krate.name, status);
```

Comment:
This would probably be better by return a result all the way up to `main()`, but this should work fine for our purposes.